src/edu/washington/escience/myria/api/encoding/CreateFunctionEncoding.java

```diff
@@ -5,11 +5,14 @@
 
 import java.util.Set;
 
+import edu.washington.escience.myria.Schema;
+
 /**
  *
  */
 public class CreateFunctionEncoding extends MyriaApiEncoding {
   @Required public String functionName;
```

Comment:
A bit of a nitpick: is it necessary for all these fields to have the prefix `function`? This isn't a pattern we follow elsewhere, and unless it disambiguates something, I feel it just adds noise.